balancer-js/src/modules/joins/joins.module.integration.spec.ts

```diff
@@ -6,16 +6,17 @@ import hardhat from 'hardhat';
 import {
   BalancerSDK,
   BalancerTenderlyConfig,
-  Network,
   GraphQLQuery,
   GraphQLArgs,
+  Network,
 } from '@/.';
 import { BigNumber, parseFixed } from '@ethersproject/bignumber';
 import { Contracts } from '@/modules/contracts/contracts.module';
-import { forkSetup, getBalances } from '@/test/lib/utils';
+import { accuracy, forkSetup, getBalances } from '@/test/lib/utils';
 import { ADDRESSES } from '@/test/lib/constants';
 import { Relayer } from '@/modules/relayer/relayer.module';
 import { JsonRpcSigner } from '@ethersproject/providers';
+import { SimulationType } from '../simulation/simulation.module';
 
 dotenv.config();
 
```

Comment:
Would be useful to add some comments with a brief explainer of what we're actually testing here - e.g. using `VaultModel` simulation type. 
I think we can also remove the Tenderly config?